### PR TITLE
feat(uapi): enforce MFA MTA-2267

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: console/my-account/how-to/enforce-mfa-for-organization.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+dates:
+  validation: 2023-06-12
+  posted: 2023-06-12
+categories:
+  - console
+---
+
+Multifactor authentication (MFA) provides extra layers of security. As well as your password (the first factor), other factors (such as a MFA app) are required to gain access to your account. Even in case of leaked password, your account is protected.
+
+You can acitvate the **enforce MFA** feature to ensure all users in your Organization set up MFA. This helps ensure the security of your Organization across all users.  or an Organization you have  have Multifactor Authentication enabled on your Scaleway account for heightened security. Once enabled, an additional security, code generated by your phone, is required to access your account.
```
Comment (RoRoJ) [inline console/my-account/how-to/enforce-mfa-for-organization.mdx:18]: this sentence seems to have gone kablooey

File: console/my-account/how-to/enforce-mfa-for-organization.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Multifactor authentication provides extra layers of security. As well as your password (the first factor), other factors (such as a MFA app) are required to gain access to your account. Even in case of leaked password, your account is protected.
```
Comment (RoRoJ) [inline console/my-account/how-to/enforce-mfa-for-organization.mdx:20]: This paragraph is a repeat of the first one. Although now I'm wondering if this whole file isn't supposed to be here?

File: console/my-account/how-to/enforce-mfa.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+<Macro id="iam-requirements" />
+
+## How to enforce MFA
+
+1. Click the name of your Organization in the top right drop-down menu. The Organization dashboard displays.
+2. Go to the **Settings** tab.
+3. Scroll to the **Organization Multifactor Authentication (MFA)** section.
+4. Click **Enforce MFA**.
+5. Write **ENFORCE** in the box.
```
Comment (RoRoJ) [inline console/my-account/how-to/enforce-mfa.mdx:27]:
```suggestion
5. Type **ENFORCE** in the box.
```

File: console/my-account/how-to/enforce-mfa.mdx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    If, however, at least one of the users does not have MFA enabled, you must send them an [MFA reminder](#how-to-send-an-mfa-reminder).
+
+    <Message type="important">
+      If you invite users who are not yet a part of your Organization, they must enable MFA before joining. <br /><br />
+      Users who are new to Scaleway must follow the [How to activate a new account after being invited to an Organization with enforced MFA](/console/my-account/how-to/accept-invitation-new-scw-account) procedure.
+    </Message>
+
+## How to send an MFA reminder
+
+1. Click IAM in the top right drop-down menu. The IAM dashboard displays.
```
Comment (RoRoJ) [inline console/my-account/how-to/enforce-mfa.mdx:44]:
```suggestion
1. Click **IAM** in the top right drop-down menu. The IAM dashboard displays.
```

File: console/my-account/how-to/enforce-mfa.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+2. Go to the **Users** tab.
+3. Check the boxes to select all users that appear to have MFA disabled.
```
Comment (RoRoJ) [inline console/my-account/how-to/enforce-mfa.mdx:46]:
```suggestion
3. Check the boxes to select all users who appear to have MFA disabled.
```

File: console/my-account/how-to/use-2fa.mdx [hunk 6]
```diff
@@ -36,19 +36,37 @@ Download the app of your choice and install it onto your smartphone.
...
 4. Enter the 6-digit code given by your app into the pop-up box, and click **Submit**. A pop-up displays:
     <Lightbox src="scaleway-2FA-popup-2.webp" alt="" />
-5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way you can regain access to your account without your MFA app. 
+5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way you can regain access to your account without your MFA app.
 
 MFA is now enabled. Next time you log in to your Scaleway account, you will be prompted for a MFA code. Use your MFA app to get the code and enter it into the Scaleway console.
 
+## How to update MFA
+
+If you no longer have access to the device in which you set up your MFA, you can update your MFA information and set up via the Scaleway console.
```
Comment (RoRoJ) [inline console/my-account/how-to/use-2fa.mdx:49]:
```suggestion
If you no longer have access to the device in which you set up your MFA, you can update your MFA information and setup via the Scaleway console.
```

File: console/my-account/how-to/use-2fa.mdx [hunk 7]
```diff
@@ -36,19 +36,37 @@ Download the app of your choice and install it onto your smartphone.
...
+
+<Message type="requirement">
+  - You have [enabled MFA](#how-to-enable-mfa)
+</Message>
+
+1. Click to open the Organization drop-down menu in the top-right corner of the console, and select **My Profile**.
+2. Scroll to the bottom of your profile page, and click **Update MFA**. A pop-up displays.
+3. Enter the code shown on the pop-up into your MFA app, or alernatively, scan the QR code into your app.
+    Your app sets up MFA for your Scaleway account, and displays a 6-digit code.
+4. Enter the 6-digit code given by your app into the pop-up box, and click **Submit**. A pop-up displays:
```
Comment (RoRoJ) [inline console/my-account/how-to/use-2fa.mdx:59]: Should there be an image here? Or else no colon I think

File: console/my-account/how-to/use-2fa.mdx [hunk 8]
```diff
@@ -36,19 +36,37 @@ Download the app of your choice and install it onto your smartphone.
...
+5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way you can regain access to your account without your MFA app.
+
 ## How to disable MFA
 
+<Message type="important">
+  You cannot disable MFA because you are a member of one or more Organizations where MFA is enforced. If you wish to disable MFA, you must first leave these Organizations. If you do not know which of your Organizations enforce MFA, follow the procedure below until step 2. The Organizations will be listed in the **Disable MFA** pop-up.
```
Comment (RoRoJ) [inline console/my-account/how-to/use-2fa.mdx:65]: Should this message say something more like "You cannot disable MFA **if** you are a member of" bla bla?

File: console/my-account/how-to/enforce-mfa.mdx [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+      You can only enforce MFA if all users in your Organization have already set up MFA on their accounts.
+    </Message>
+6. Click **Enforce MFA**.
+    If 100% of users in your Organization already have MFA enabled, a confirmation message appears on the top right corner of the screen.
+
+    If, however, at least one of the users does not have MFA enabled, you must send them an [MFA reminder](#how-to-send-an-mfa-reminder).
+
+    <Message type="important">
+      If you invite users who are not yet a part of your Organization, they must enable MFA before joining. <br /><br />
+      Users who are new to Scaleway must follow the [How to activate a new account after being invited to an Organization with enforced MFA](/console/my-account/how-to/accept-invitation-new-scw-account) procedure.
```
Comment (nerda-codes) [inline console/my-account/how-to/enforce-mfa.mdx:39]:
```suggestion
      Users who are new to Scaleway must follow the [procedure to activate a new account after being invited to an Organization with enforced MFA](/console/my-account/how-to/accept-invitation-new-scw-account).
```

File: console/my-account/how-to/enforce-mfa.mdx [hunk 10]
```diff
@@ -0,0 +1,70 @@
+---
+meta:
+  title: How to enforce Multifactor Authentication for an Organization
+  description: This page explains how to enforce Multifactor Authentication for an Organization
+content:
+  h1: How to enforce Multifactor Authentication for an Organization
+  paragraph: This page explains how to enforce Multifactor Authentication for an Organization
+tags: authentication 2FA two-factor two multi factor security google authenticator authenticator
```
Comment (bene2k1) [inline console/my-account/how-to/enforce-mfa.mdx:8]:
```suggestion
tags: authentication 2FA two-factor two multi factor security google authenticator authenticator mfa
```

**Review by bene2k1 — APPROVED** [review]

File: console/my-account/how-to/enforce-mfa.mdx [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+1. Click the name of your Organization in the top right drop-down menu. The Organization dashboard displays.
+2. Go to the **Settings** tab.
+3. Scroll to the **Organization Multifactor Authentication (MFA)** section.
+4. Click **Stop enforcing MFA**.
+    <Message type="requirement">
+      Disabling this feature means that users in this Organization will no longer be required to have MFA set up on their accounts.
+    </Message>
+5. Type **STOP**.
+6. Click **Stop enforcing MFA** to confirm.
+    All users in your Organization will receive an informative email saying MFA is no longer enforced in the Organization.
```
Comment (crlptl) [inline console/my-account/how-to/enforce-mfa.mdx:63]: The mail is sent only to the owner (just tested it)

File: console/my-account/how-to/use-2fa.mdx [hunk 12]
```diff
@@ -36,19 +36,37 @@ Download the app of your choice and install it onto your smartphone.
...
+5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way you can regain access to your account without your MFA app.
 
 MFA is now enabled. Next time you log in to your Scaleway account, you will be prompted for a MFA code. Use your MFA app to get the code and enter it into the Scaleway console.
 
+## How to update MFA
+
+If you no longer have access to the device in which you set up your MFA, you can update your MFA information and set up via the Scaleway console.
+
+<Message type="requirement">
+  - You have [enabled MFA](#how-to-enable-mfa)
```
[the remark this answers is not in the document]
Comment (crlptl) [reply console/my-account/how-to/use-2fa.mdx:52]: I +1 myself from the past

**Pull request merged**